### ISSUE TITLE
Undefined property: stdClass::$caption error message in 2.7.0-pl

### ISSUE THREAD

**Issue by jonleverrier** [Jako/ImagePlus#71]
Since upgrading to 2.7.0-pl i've been getting these error messages in my log
```
(WARN @ /home/user/public/project/core/components/imageplus/model/cropengines/PhpThumbOf.php : 170) PHP notice: Undefined property: stdClass::$credits
```
```
(WARN @ /home/user/public/project/core/components/imageplus/model/cropengines/PhpThumbOf.php : 169) PHP notice: Undefined property: stdClass::$caption
```
Is this a user error or something related to ImagePlus?

**Comment by Jako** [comment]
Seams so

**Comment by Jako** [comment]
Will be fixed in 2.8.0

**Comment by Jako** [comment]
You could you try the 2.8.0-rc1: https://github.com/Jako/ImagePlus/raw/master/_packages/imageplus-2.8.0-rc1.transport.zip
